src/constants.mjs

```diff
@@ -0,0 +1,293 @@
+'use strict';
+
+// This is the base-path of API docs within the https://nodejs.org Website
+export const DOC_WEB_BASE_PATH = '/docs/';
+
+// This is the base URL of the MDN Web documentation
+export const DOC_MDN_BASE_URL = 'https://developer.mozilla.org/en-US/docs/Web/';
+
+// This is the base URL for the MDN JavaScript documentation
+export const DOC_MDN_BASE_URL_JS = `${DOC_MDN_BASE_URL}JavaScript/`;
+
+// This is the base URL for the MDN JavaScript Primitives documentation
+export const DOC_MDN_BASE_URL_JS_PRIMITIVES = `${DOC_MDN_BASE_URL_JS}Data_structures`;
+
+// This is the base URL for the MDN JavaScript Global Objects documentation
+export const DOC_MDN_BASE_URL_JS_GLOBALS = `${DOC_MDN_BASE_URL_JS}Reference/Global_Objects/`;
+
+// These are YAML keys from the Markdown YAML Metadata that should always be Arrays
+export const DOC_API_YAML_KEYS_ARRAYS = [
+  'added',
+  'napiVersion',
+  'deprecated',
+  'removed',
+  'introduced_in',
+];
+
+// These are YAML keys from the Markdown YAML Metadata that should be
+// removed and appended to the `update` key
+export const DOC_API_YAML_KEYS_UPDATE = [
+  'added',
+  'removed',
+  'deprecated',
+  'introduced_in',
+  'napiVersion',
+];
+
+// These are the YAML Types that should be considered as Navigation Entries
+export const DOC_API_YAML_KEYS_NAVIGATION = ['class', 'module', 'global'];
+
+export const DOC_API_SLUGS_REPLACEMENTS = [
+  { to: 'nodejs', from: /node.js/ }, // Replace node.js
+  { to: '-and-', from: /&/ }, // Replace &
+  { to: '-', from: /[/_,:;\\ ]/g }, // Replace /_,:;\. and whitespace
+];
+
+export const DOC_TYPES_MAPPING_PRIMITIVES = {
```

Comment:
```suggestion

// ↓↓↓ Mapping of Doc Types to their respective links ↓↓↓

export const DOC_TYPES_MAPPING_PRIMITIVES = {
```

Reply:
What is this comment about? 👀

Reply:
This is the explanation of the constants just below. At least that's my understanding of it. Because without comment there are just constants.